### PR TITLE
work on fetch API and get quotes

### PR DESCRIPTION
**Pull request by jaimevillegas**
Hi there!

I have made the following changes:

- Add `api.js` on folder `components` with functionality to fetch data from the NinjaAPI and create a React component. :zap:
- Add the `Api` component to `index.js`. :heavy_plus_sign:
- Add some styling to the quote. :art:

Thank you for taking the time to review my code!

### CODE REVIEW

File: src/components/api.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from 'react';
+
+const baseURL = 'https://api.api-ninjas.com/v1/quotes';
+export default function ApiNinja() {
+  const [data, setData] = useState(null);
+  const [loading, setLoading] = useState(false);
+
```
Comment (Wes-Isaac) [inline src/components/api.js]: - [x] Great job on adding a loading state, it's also one of the requirements to add an error state if errors come up while sending requests, let's show a text that says something went wrong 👍.

File: src/components/api.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from 'react';
+
+const baseURL = 'https://api.api-ninjas.com/v1/quotes';
```
Comment (Wes-Isaac) [inline src/components/api.js]: - [x] [_OPTOINAL_] This is optional but since we are using this component to show a quote it's better to name it Quote.js

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @jaimevillegas,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Professionally documented your work 💯.
- Linters setup properly 👍.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/api.js (hunk 1 above)
Comment (jaimevillegas) [reply src/components/api.js]: Thank you for the comment! I have added an error state now.

File: src/components/api.js (hunk 2 above)
Comment (jaimevillegas) [reply src/components/api.js]: It is a good suggestion! I have changed the file name and the function name for the one that you suggested. Thank you!

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @jaimevillegas ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- Good use of GitFlow :+1: 
- Good job adding the error state :+1: 
## Optional suggestions :small_airplane: 
- [OPTIONAL] I suggest setting up linter checks properly by adding [webhint](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml#:~:text=webhint%3A,%3A%20npx%20hint%20.) test as per the general requirements  :recycle: 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**